src/phases/egg-hatch-phase.ts

```diff
@@ -285,14 +285,14 @@ export class EggHatchPhase extends Phase {
     this.canSkip = false;
     this.hatched = true;
     if (this.evolutionBgm) {
-      SoundFade.fadeOut(this.scene, this.evolutionBgm, Utils.fixedInt(100));
+      SoundFade.fadeOut(this.evolutionBgm, Utils.fixedInt(100));
```

Comment:
```suggestion
      SoundFade.fadeOut(gScene, this.evolutionBgm, Utils.fixedInt(100));
```
(see comment in evolution phase)